dashboards/use.libsonnet

```diff
@@ -149,8 +149,7 @@ local g = import 'grafana-builder/grafana.libsonnet';
           g.panel('Disk Utilisation') +
           g.queryPanel(
             |||
-              max ((node_filesystem_size{%(fstypeSelector)s} - node_filesystem_avail{%(fstypeSelector)s})
-              / node_filesystem_size{%(fstypeSelector)s}) by (namespace, %(podLabel)s, device)
+              node:node_filesystem_usage:
```

Comment:
This is a copy & paste typo. It needs to be `node:node_filesystem_avail:`

Reply:
ah, right, I renamed these, good catch :+1:

Reply:
wait, this panel used to show disk utilisation, not how much disk space is free (I just remember to have tested this on Friday). Are you sure to flip it?

Reply:
I just saw that you extracted these queries into recording rules but now used the same recording rule.

Reply:
yes, that is correct, both
https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/76/files#diff-4d6886e4c5a0694f485442b13839ceafR136
and
https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/76/files#diff-3f0aaae366dd9f513f22b416d7839cbdR152
refer to the same recording rule `node:node_filesystem_usage:`.

The whole dashboard in `node.libsonnet` is redundant with the node dashboard in `use.libsonnet`. @brancz suggested to remove the former at all in a subsequent PR.

I added `node:node_filesystem_avail:` as it may make sense to be used in an alert as requested here: https://github.com/openshift/cluster-monitoring-operator/issues/74